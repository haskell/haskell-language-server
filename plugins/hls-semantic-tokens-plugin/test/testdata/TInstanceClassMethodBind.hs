module TInstanceClassMethodBind where


data Foo = Foo Int
instance Eq Foo where
    (==) = undefined
