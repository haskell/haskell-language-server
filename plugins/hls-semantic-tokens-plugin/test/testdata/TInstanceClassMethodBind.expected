4:6-9 TTypeConstructor "Foo"
4:12-15 TDataConstructor "Foo"
4:16-19 TTypeConstructor "Int"
5:10-12 TClass "Eq"
5:13-16 TTypeConstructor "Foo"
6:5-9 TClassMethod "(==)"
6:12-21 TVariable "undefined"
